specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/account.json

```diff
@@ -0,0 +1,964 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagement",
+    "version": "2023-05-15-preview",
```

Comment:
Please check the breaking changes comment: [https://github.com/Azure/azure-rest-api-specs/pull/23988#issuecomment-1634101907](https://github.com/Azure/azure-rest-api-specs/pull/23988#issuecomment-1634101907). ARM will review your PR changes once breaking changes are resolved.